inference-engine/tests/functional/plugin/shared/CMakeLists.txt

```diff
@@ -20,6 +20,15 @@ addIeTarget(
             ${CMAKE_CURRENT_SOURCE_DIR}/src
         ADD_CPPLINT
         DEVELOPER_PACKAGE
+        INCLUDES
+            PUBLIC
+                ${PUBLIC_HEADERS_DIR}
+                ${IE_MAIN_SOURCE_DIR}/src/vpu/graph_transformer/include
```

Comment:
why shared tests include VPU specifics headers?

Reply:
Thanks, fixed. ProducrConfig MR number is 574